### PR TITLE
:sparkles: send price

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: ui/ui-expenses/src/components/header/header.jsx [hunk 1]
```diff
@@ -7,9 +7,7 @@ const Header = ({ name, avatar, title }) => (
   <div className={styles.header}>
     <img src={avatar} alt={name} className={styles.avatar} />
     <h1 className={styles.title}>{title}</h1>
-    {
-      title === 'envoi' && <Send />
-    }
+    <Send />
```
Comment (delphinemillet) [inline ui/ui-expenses/src/components/header/header.jsx]: The purpose of this button is to send recorded expenses to the acocuntant.
To add an expense to the list, there will be a button at the bottom of the page.
That's why there was a condition on this component

File: ui/ui-expenses/src/screens/list/list.reactions.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ export const submit = ({ payload }, store, { window, http }) => {
           reader2.onload = async (readerE) => {
             http('IMAGES').post('/api/images', {
               image: readerE.target.result.replace(/data:.*;base64,/, ''),
-              user: 'fabien',
+              user: store.data.profile.get().name,
```
Comment (delphinemillet) [inline ui/ui-expenses/src/screens/list/list.reactions.js]: better 😄

File: ui/ui-expenses/src/components/header/header.jsx (hunk 1 above)
Comment (fabienjuif) [reply ui/ui-expenses/src/components/header/header.jsx]: **acocuntant**

Comment (delphinemillet) [reply ui/ui-expenses/src/components/header/header.jsx]: it was late !!!
*accountant*

Comment (guillaumecrespel) [reply ui/ui-expenses/src/components/header/header.jsx]: i'm ok with you @delphinemillet !
I let you do the design ?
But we can actually think of a generic button component with just send event ?

Comment (delphinemillet) [reply ui/ui-expenses/src/components/header/header.jsx]: Here I'm not sure we need a generic component: design, position in the screen and purpose of the action totally different for the 2 buttons.
ok for the design, I take the point !

Comment (fabienjuif) [reply ui/ui-expenses/src/components/header/header.jsx]: @delphinemillet the generic button could be used into expenses AND invoices.
If you are ok, maybe we should do a `packages` with all our common components ?
 - Maybe `packages/ui-components` ?

Comment (guillaumecrespel) [reply ui/ui-expenses/src/components/header/header.jsx]: new issues for this point : #26

Comment (delphinemillet) [reply ui/ui-expenses/src/components/header/header.jsx]: ok, I didn't understood what you meant 😵 
So have generic components in a common package is a good idea 👍

File: ui/ui-expenses/src/screens/prices/prices.reactions.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+export const send = ((action, store, { http }) => {
```
Comment (delphinemillet) [inline ui/ui-expenses/src/screens/prices/prices.reactions.js]: According to what I have understand about the user flow, the button send on the header, visible only on the screen `list`, send the list of the pending expenses.
When you create a new expense, you add it to the list but don't send it directly to the accountant (so the list need to be persisted)

File: ui/ui-expenses/src/components/header/header.jsx (hunk 1 above)
Comment (delphinemillet) [reply ui/ui-expenses/src/components/header/header.jsx]: @guillaumecrespel do you agree to put back the condition on the display of `Send` ? (initial way or another if you prefer)

File: ui/ui-expenses/src/screens/prices/prices.reactions.js (hunk 3 above)
Comment (delphinemillet) [reply ui/ui-expenses/src/screens/prices/prices.reactions.js]: You can add a reaction `add` that add the created expense to the list of pending expenses, and the button `valider` (style coming later) in the screen if you are in the mood 😄

**Comment by guillaumecrespel** [comment]
I've add button to add price and revert the header.
I've add a "notification" response

File: ui/ui-expenses/src/screens/prices/prices.reactions.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+export const setPrice = (({ payload }, store) => {
+  store.ui.price.set(payload)
+})
+
+export const response = ((action, store) => {
+  store.ui.response.set('price adding 👏')
+})
+
+export const error = (({ payload }, store) => {
+  store.ui.response.set('an error occured 🤮')
```
Comment (delphinemillet) [inline ui/ui-expenses/src/screens/prices/prices.reactions.js]: the character is not found 😞

File: ui/ui-expenses/src/screens/prices/prices.reactions.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      { credentials: 'include' },
+    )
+})
+
+export const setPrice = (({ payload }, store) => {
+  store.ui.price.set(payload)
+})
+
+export const response = ((action, store) => {
+  store.ui.response.set('price adding 👏')
```
Comment (delphinemillet) [inline ui/ui-expenses/src/screens/prices/prices.reactions.js]: `price added` is better ! But inconsistent with the rest of the app which is in french 😅

**Pull request merged**